frontend/src/components/landing/kpi/kpi.tsx

```diff
@@ -1,41 +1,68 @@
 import { APP_CONTENT } from "@/utils";
 import styles from "./kpi.module.css";
+import { API_ENDPOINTS, apiClient } from "@/services";
+import { useQuery } from "@tanstack/react-query";
 
 type TKPIS = {
-  figure: string;
+  figure?: number;
   label: string;
 }[];
 
-const KPIs: TKPIS = [
-  {
-    figure: "15",
-    label: APP_CONTENT.homepage.kpi.publishedAIModels,
-  },
-  {
-    figure: "120",
-    label: APP_CONTENT.homepage.kpi.totalUsers,
-  },
-  {
-    figure: "100",
-    label: APP_CONTENT.homepage.kpi.humanFeedback,
-  },
-  {
-    figure: "15",
-    label: APP_CONTENT.homepage.kpi.acceptedPrediction,
-  }
-  // ,
-  // {
-  //   figure: "86",
-  //   label: "Mappers",
-  // },
-];
+type TKPIResponse = {
+  total_accepted_predictions: number;
+  total_feedback_labels: number;
+  total_models_published: number;
+  total_registered_users: number;
+};
+
+const fetchKPIStats = async (): Promise<TKPIResponse> => {
+  const { data } = await apiClient.get(API_ENDPOINTS.GET_KPI_STATS);
+  return data;
+};
 
 const Kpi = () => {
+  const { data, isLoading,isError,error} = useQuery({
+    queryKey: ["kpis"],
+    queryFn: fetchKPIStats,
+  });
+
+  if (isError) {
+    return (
+      <section className={styles.kpiContainer}>
+        <div>
+          <h2>Error fetching KPI Stats</h2>
+          <p>{(error as Error)?.message || 'An unknown error occurred.'}</p>
+        </div>
+      </section>
+    );
+  }
+
+  const KPIs: TKPIS = [
+    {
+      figure: data?.total_models_published ?? 0,
+      label: APP_CONTENT.homepage.kpi.publishedAIModels,
```

Comment:
we can add some format for the numbers

Reply:
Do you mean like 12k, 20k etc @omranlm ?

Reply:
like 1,546 instead of 1546... no worries, I will do that with other text changes ...

Reply:
Oh, okay. Thanks!